src/lib/loaders/api/index.js

```diff
@@ -17,33 +17,49 @@ export default requestIDs => ({
   /**
    * The Delta loaders produced by this factory _will_ cache all responses to memcache.
    */
-  deltaLoaderWithoutAuthenticationFactory: apiLoaderWithoutAuthenticationFactory(delta, "delta", {
-    requestIDs,
-  }),
+  deltaLoaderWithoutAuthenticationFactory: apiLoaderWithoutAuthenticationFactory(
+    delta,
+    "delta",
+    {
+      requestIDs,
+    }
+  ),
```

Comment:
@alloy, @orta - Applying Prettier over the codebase made these previous one-liners jump out, as well as where they're used in the code. 

This is debatable, but I think one way to clean up some of our loader use throughout MP is to refactor `apiLoaderWithoutAuthenticationFactory` (and other such factories) into a function that takes a single configuration object, rather than a function that takes multiple unnamed parameters. 

For example:
```javascript
export const apiLoaderWithoutAuthenticationFactory = (api, { 
  apiName,
  globalAPIOptions = {}
}) => {
```
And then later:
```javascript
export default requestIDs => ({
  gravityLoaderWithoutAuthenticationFactory: apiLoaderWithoutAuthenticationFactory(delta, {
    apiName: "delta",
    globalAPIOptions: {
      requestIDs,
    }
  }),
```
Which further down the tree in standard usage would look like:
```javascript
similarGenesLoader: gravityLoader(id => `gene/${id}/similar`, {
  apiOptions: {
    headers: true
  }
})
```
rather than 
```javascript
similarGenesLoader: gravityLoader(
  id => `gene/${id}/similar`,
  {},
  { headers: true }
),
```

At a glance unnamed parameters make API use less informative, and also leaves a lot of empty data around as sometimes the last parameter is needed but the intermediate two aren't. I've found that Relay often suffers from this as well. 

When everything is on one line this isn't as noticable, but now that we're at 80 column width it most definitely reduces readability.